README.md

```diff
@@ -0,0 +1,22 @@
+# Adafruit WebSerial ESPTool
+
+JavaScript package to install firmware on ESP devices via the browser using WebSerial.
+
+## Used by
+
+- [Adafruit WipperSnapper](https://learn.adafruit.com/quickstart-adafruit-io-wippersnapper)
+- [Adafruit CircuitPython Installer](https://circuitpython.org/downloads)
+
+## Local development
+
+- Clone this repository.
+- Install dependencies with `npm`
+- Run `script/develop`
+- Open http://localhost:5004/
+
+## Origin
+
+This project was originally written by [Melissa LeBlanc-Williams](https://github.com/makermelissa). [Nabu Casa](https://www.nabucasa.com) ported the code over to TypeScript and in March 2022 took over maintenance from Adafruit. In July 2022, the Nabucasa stopped maintaining the project in favor of an official, but very early release of Espressif's [esptool-js](https://github.com/espressif/esptool-js/). Due to the instability of the tool, Adafruit updated their fork with Nabucasa's changes in November 2022 and took over maintenance once again.
```

Comment:
This is out of date now.

Reply:
Update this history to say we went back to the Espressif version (or just remove the detailed history).